README.md

```diff
@@ -106,6 +76,79 @@ These steps you need to do once to setup the project.
   ```bash
   pipenv install
   ```
+  
+An alternative to ```pipenv install``` is to use the ```make setup``` command, which will install the environment, 
+apply a patch to the multihash package to support md5, create the volumes needed by the Postgres and MinIO containers
+and run those containers. ```Make setup``` assume a standard local installation with a dev environment.
+  
+### Setting up the local database
+
+The service use two other services to run, Posgres with a PostGIS extension and S3. 
+For local development, we recommend using the services given through the docker-compose.yml file, which will 
+instantiate a Postgres container and a [MinIO](https://www.min.io/) container which act as a local S3 replacement.
+
+If you used the ```make setup``` command during the local environment creation, those two services 
+should be already be up. You can check with 
+
+  ```bash
+  docker ps -a
+  ```
+
+which should give you a result like this : 
+  ```
+  CONTAINER ID   IMAGE                  COMMAND                   CREATED        STATUS                      PORTS                     NAMES
+  a63582388800   minio/mc               "/bin/sh -c '\n  set …"   39 hours ago   Exited (0) 40 seconds ago                             service-stac_s3-client_1
+  33deededf690   minio/minio            "/usr/bin/docker-ent…"    39 hours ago   Up 41 seconds               0.0.0.0:9090->9000/tcp    service-stac_s3_1
+  d158be863ac1   kartoza/postgis:12.0   "/bin/sh -c /docker-…"    39 hours ago   Up 41 seconds               0.0.0.0:15432->5432/tcp   service-stac_db_1
+  ```
+
+As you can see, MinIO is using two containers, one is the local S3 server, the other is a S3 client used to initialise
+some variables to allowe MinIO to run smoothly (and exited normally afterwards). You should also have a postGIS container.
```

Comment:
The minio client is used to set the download policy to the bucket which allows anonymous download.

Reply:
Thanks, I have corrected the information.